aws/logs_monitoring/trace_forwarder/scripts/run_tests.sh

```diff
@@ -12,5 +12,5 @@ set -e
 cd $(dirname "$0")/..
 
 docker build -t datadog-go-layer . --build-arg runtime=python:3.7
-docker run datadog-go-layer go test -v ./...
+docker run --rm -e UPDATE_SNAPSHOTS=$UPDATE_SNAPSHOTS datadog-go-layer go test -v ./...
```

Comment:
Just want to double check this will update the local filesystem when using UPDATE_SNAPSHOTS=true. It might not work if we are copying files into the container, rather than using a local volume.

Reply:
oh good call, I'll remove and just leave the remove flag .